index.html

```diff
@@ -195,14 +195,21 @@ <h2>Contact me</h2>
                 <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
                 </p>
                 <form action="https://formspree.io/f/xyyvwqrd" method="POST">
-                    <label for="full-name" hidden>put your full name</label>
-                    <input type="text" required name="full-name" id="full-name" placeholder="full name" max-length="30">
+                    <label for="full-name">
+                        <span hidden>put your full name</span>
+                        <input type="text" required name="full-name" id="full-name" placeholder="full name" max-length="30">
+                    </label>
                     
-                    <label for="email" hidden>put your Email</label>
-                    <input type="email" id="email" required name="Email" placeholder="email" max-length="30">
+                    <label for="email">
+                        <span hidden>put your Email</span>
+                        <input type="email" id="email" required name="Email" placeholder="email" max-length="30">
+                        <span class="erorr-massege" aria-live="polite">the Email should be in lower cass</span>
```

Comment:
[Optional]
- [ ] If you can fix the typo in the error message, your project will look more professional.
```suggestion
                        <span class="erorr-massege" aria-live="polite">the Email should be in lower case</span>
```